### PR TITLE
Added image component

### PR DESCRIPTION
**Pull request by mbos2**
Added image component with an option to make it animated n hover and focus

### CODE REVIEW

File: src/lib/styles/mixins.sass [hunk 1]
```diff
@@ -31,3 +31,10 @@
   border-radius: var(--radius)
   box-shadow: 1px 1px 5px 0 rgba(0, 0, 0, 0.02), 1px 1px 15px 0 rgba(0, 0, 0, 0.03)
   transition: transform var(--transition-normal), background-color var(--transition-normal), box-shadow var(--transition-slow)
+
+@mixin image-hover-animation
+  transition: transform var(--transition-fast) ease-in-out
+  &:hover
+    transform: scale(1.1)
+  &:focus
```
Comment (Knochenmark) [inline src/lib/styles/mixins.sass:39]: I would drop focus, hover should suffice

File: src/lib/styles/mixins.sass [hunk 2]
```diff
@@ -31,3 +31,10 @@
   border-radius: var(--radius)
   box-shadow: 1px 1px 5px 0 rgba(0, 0, 0, 0.02), 1px 1px 15px 0 rgba(0, 0, 0, 0.03)
   transition: transform var(--transition-normal), background-color var(--transition-normal), box-shadow var(--transition-slow)
+
+@mixin image-hover-animation
```
Comment (Knochenmark) [inline src/lib/styles/mixins.sass:35]: maybe rename to something shorter? `hover-scale` or something like that? technically it's not bound to an imge like that

File: src/lib/styles/mixins.sass (hunk 1 above)
Comment (mbos2) [reply src/lib/styles/mixins.sass]: Done

File: src/lib/styles/mixins.sass (hunk 2 above)
Comment (mbos2) [reply src/lib/styles/mixins.sass]: Done

**Review by Knochenmark — APPROVED** [review]

**Pull request merged**